addons/prometheus/prometheus.yaml

```diff
@@ -206,7 +206,7 @@ spec:
                   insecureSkipVerify: true
         prometheusSpec:
           image:
-            tag: v2.19.2
+            tag: v2.22.2
```

Comment:
Anything noteworthy in these releases in terms of notes for our users?

Reply:
Lots of new fixes and features, but I didn't see anything that looked like it should be surfaced in KBA release notes.